autonomy/cli/helpers/chain.py

```diff
@@ -79,7 +79,9 @@ def get_ledger_and_crypto_objects(
     chain_config = ChainConfigs.get(chain_type=chain_type)
     if chain_config.rpc is None:
         raise click.ClickException(
-            f"RPC cannot be `None` for chain config; chain_type={chain_type}"
+            f"RPC cannot be `None` for chain config\n"
+            f"Please set the environment variable for {chain_type.value} "
+            f"using `{ChainConfigs.get_rpc_env_var(chain_type)}` environment variable"
```

Comment:
```suggestion
            f"Chain configuration for {chain_type.value} was not found.\n"
            f"Please make sure you have set the `{ChainConfigs.get_rpc_env_var(chain_type)}` environment variable."
```

Reply:
The entire chain is not missing, just the RPC url is missing